src/Components/Activities/Activities.js

```diff
@@ -1,10 +1,15 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import './Activities.css';
-import { Link } from 'react-router-dom';
+import { Link, useRouteMatch } from 'react-router-dom';
 
-const Activities = ({ activities, updateActivity }) => {
+const Activities = ({ activities, updateActivity, setFeeling }) => {
   const [currentActivityIndex, setCurrentActivityIndex] = useState(0);
 
+  const { params } = useRouteMatch('/what-should-you-do/:feeling');
```

Comment:
Pretty sure I'm following, but would love to hear more tomorrow! Haven't used useRouteMatch before.

Reply:
Really cool! So this is setting the feeling in activities based on the match object from the Link that they set when they click on the feeling, which triggers the filterActivites in the useEffect in App, and then renders the proper activities!? I'm pretty geeked on this Amber this is awesome

Reply:
+100000 this is SUPER dope Amber! Great work!!